panda-utilities/src/test/java/org/panda_lang/utilities/commons/iterable/ArrayIterableTest.java

```diff
@@ -0,0 +1,23 @@
+package org.panda_lang.utilities.commons.iterable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Iterator;
+
+class ArrayIterableTest {
+
+    @Test
+    void next_should_moves_to_the_next_element_and_return_it() {
```

Comment:
Should `move`, not `moves` (https://data.grammarbook.com/blog/uncategorized/when-to-add-s-to-a-verb-2/)